### PR TITLE
Add "targetNamespace" support

### LINKED ISSUE #34: Add support for setting "targetNamespace" in client

### PR DESCRIPTION
**Pull request by sfitts**
Can set for the client which will get added to requests

Fixes #34

### CODE REVIEW

**Review by AlexBlumer — APPROVED** [review]
LGTM

**Review by fhcarter — APPROVED** [review]

**Comment by fhcarter** [comment]
probably need to bump the version in build.gradle so it can by Pypi'd appropriately

**Comment by sfitts** [comment]
> probably need to bump the version in build.gradle so it can by Pypi'd appropriately. Since new method, goes to next minor version (rather than just bug fix increment).

Yep -- I tend to do that post review, but thanks for the reminder (because I have forgotten it ;) ).

File: build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ ext {
     }
 
     if (!project.rootProject.hasProperty('vantiqSdkVersion')) {
-        vantiqSdkVersion = '1.2.4'
+        vantiqSdkVersion = '1.2.5'
```
Comment (fhcarter) [inline build.gradle:15]: Seems like this should move to 1.3.0 since we're adding an entry point.  I don't think anyone's using these much, but...

Comment (sfitts) [reply build.gradle:15]: Seemed like a minor addition, not really an API change.  I've already published as `1.2.5`, so unless there are strong objections, I'm going to leave as is.

Comment (fhcarter) [reply build.gradle:15]: ok

**Pull request merged**